docker-hub/roles-and-permissions.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Use roles in your organization to control who has access to content, registry, and organization management permissions.
+keywords: members, teams, organization, company, roles, access
+title: Roles and permissions
+---
+
+Organization and company owners can assign roles to individuals giving them different permissions in the organization. This section is for owners who want to learn about the defined roles and their permission scopes.
+
+## Roles
+
+When you invite users to your organization on Docker Hub, you assign a role. A role is a collection of permissions. Roles define access to perform actions like creating repositories, pulling images, publishing extensions, or managing repositories.
+
+The following roles are available to assign:
+
+- **Member** - Non-administrative role. Members can view other members that are in the same organization.
+- **Editor** - Partial administrative access to the organization. Editors can create, edit, and delete repositories. They can also edit an existing team's access permissions.
+- **Organization owner** - Full organization administrative access. Organization owners can manage organization repositories, teams, members, settings, and billing.
+- **Company owner** - In addition to the permissions of an organization owner, company owners can configure settings for their associated organizations.
+
+## Permissions
+
+The following sections outline the permissions for each role.
+
+### Content and registry permissions
+
+| Permission | Member | Editor | Organization owner | Company owner |
+|:----------------------- |:------ |:-------|:------------------ |:----------- |
+| Explore images and extensions | ✅ | ✅ | ✅ | ✅ |
+| Star, favorite, vote, and comment on content | ✅ | ✅ | ✅ | ✅ |
+| Pull images | ✅ | ✅ | ✅ | ✅ |
```

Comment:
@stephaurelio IIRC company owners wont have permissions to Pull/Push images, right  @docker/accounts ?

Reply:
Although that has been checked off in all versions of this table, so just want to verify, should this be marked ❌ @siarheikarneichyk ?

Reply:
Company owners have full permission to push/pull images.

Reply:
When it comes to repositories, companies currently have no meaning. So no, company owners cannot push/pull anything.

Reply:
We should probably remove this, and all repo permission related items as they are not impacted by RBAC (by design)

Reply:
Removed the company owner column here since their role is more relevant to the second table, and added the callout at the intro about associated orgs. Thanks @jcarter3 for this feedback!